src/pages/index.tsx

```diff
@@ -40,12 +62,16 @@ export default function Home() {
                 <p className='text-xl font-semibold'>日付を入力</p>
               </div>
               <p className='ml-4 text-sm'>※候補日時を選択</p>
-              <div>ここにカレンダー</div>
+              <Calender
+                onChange={(selectedDate: number[]) => {
+                  setValue('dates', selectedDate)
+                }}
+              />
             </div>
```

Comment:
datesもrequiredのはずなので、バリデーションをつけたいです

```
  const watchedCalender = watch('dates')
  const calenderValid = useMemo(() => {
    return watchedCalender?.length !== 0
  }, [watchedCalender])
```

みたいにバリデーションを作って、

```suggestion
            </div>
              {!calenderValid && (
                <p className='text-red-500'>カレンダーを入力してください</p>
              )}```

としてエラー文を表示するのがいいかな

Reply:
[c3d0071](https://github.com/NUTFes/chosei-chan/pull/46/commits/c3d0071ae64c87dd2bc3d90df4c6baa6e494a2ae)